### ISSUE TITLE
domain label are different from paper

### ISSUE THREAD

**Issue by jhzhang19** [VisionLearningGroup/DA_Detection#39]
Thank you for your great work. I find a question in loss function: the domain_s label is 1 and domain_t label is 0 in paper, but in the code domain label are 0 and 1. In addition,the loss function in paper is contradictory

**Comment by Shuntw6096** [comment]
I think that  `the domain_s label is 1 and domain_t label is 0` the paper mentioned is for backbone (generator), because the two alignment networks(discriminator) connect to backbone through gradient reverse, for backbone, optimization direction is minimum, source domain label is 1 we want for backbone, but discriminator use reverse optimization direction to train, so use flipped label to train), same as [GAN training](https://github.com/linxi159/GAN-training-tricks#2-a-modified-loss-function).(if I understand adversarial learning correctly)

**Comment by a21401624** [comment]
> Thank you for your great work. I find a question in loss function: the domain_s label is 1 and domain_t label is 0 in paper, but in the code domain label are 0 and 1. In addition,the loss function in paper is contradictory

I read the paper yesterday and I think I have found the same issue. The formulation (8) and (9) in the paper are inverse,  if the domain_s label is 1 and domain_t label is 0. I think the question is evident.
